vendor/vendor.json

```diff
@@ -271,6 +283,12 @@
 			"version": "v2.0.0-preview.5",
 			"versionExact": "v2.0.0-preview.5"
 		},
+		{
+			"checksumSHA1": "oBbcBm1YWxduO3qRDLI5ZOfRVvw=",
+			"path": "github.com/aws/aws-sdk-go-v2/private",
+			"revision": "db223f6d5a456d1c9dfaeb7ef865cc63e03ec2ce",
```

Comment:
same here, sorry!

Reply:
I think this is now used by many of the other API packages with the upgrade

Reply:
So if https://github.com/elastic/beats/pull/12548/files#diff-bd290170e2912d3e8694db1a151066e5R298 and https://github.com/elastic/beats/pull/12548/files#diff-bd290170e2912d3e8694db1a151066e5R266 are deleted, this PR will fail?